src/main/java/alvioneurope/gsergeev/javagroup/clientlogimporter/ExecutedCommand.java

```diff
@@ -7,9 +7,57 @@
     private String requestId;
     private Command command;
     private int timeout;
-    private  int requestLength;
+    private int requestLength;
     private String operationId;
     private int responseLength;
     private int execTime;
 
+    public ExecutedCommand(String parse)
+    {
+    	String[] parseArray = parse.split("  ");
```

Comment:
It's really important. You shouldn't declare variable separately from using. Please move it down.